<?xml version="1.0" encoding="UTF-8" ?>

<launch>
    <include file="$(find vrep_ros_teleop)/launch/teleop_mux.launch"/>

    <node pkg="floor_plane_deep" type="floor_plane_classifier.py" name="floor_plane_classifier" output="screen">
      <param name="model_dir" value="$(find floor_plane_deep)/models/models/best_model" />
      <param name="thumb_size" value="32" />

      <remap from="/floor_plane_classifier/image" to="/vrep/visionSensor"/>
    </node>
    
    <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
        <remap from="image" to="/floor_plane_classifier/image_label" />
    </node>
</launch>
